src/components/card.js

```diff
@@ -1,39 +1,83 @@
 export default class Card {
```

Comment:
* File `card.js` should be in **UpperCase** according to the checlist: `Card.js`

If the file name doesn't change on github, then you need to clear git cache

```bash

git rm -r --cached .
git add .
git commit -am 'git cache cleared'
git push

```